.goreleaser.yml

```diff
@@ -0,0 +1,40 @@
+project_name: terraform-provider-gitlab
+
+builds:
+  - binary: "terraform-provider-gitlab_{{.Tag}}"
```

Comment:
As already said elsewhere, we should take care to prefix or suffix tags with `claranet-`/`-claranet` to avoid clashes with official releases.

Reply:
We decided to use the `claranet` suffix directly in the tag name. So there will no clashes with official releases (with official tags already there in our fork).

Reply:
Yes, this was just a reminder :)